holoviews/ipython/__init__.py

```diff
@@ -183,20 +183,23 @@ def __call__(self, *args, **params):
         if hasattr(config, 'comms') and comms:
             config.comms = comms
 
-        for r in [r for r in resources if r != 'holoviews']:
-            Store.renderers[r].load_nb(inline=p.inline)
-        Renderer.load_nb(inline=p.inline)
+        if not same_cell_execution:
```

Comment:
Variable is not defined yet, you define it below.

Reply:
Oops! Well spotted.

Reply:
Fixed in [b946bbf](https://github.com/holoviz/holoviews/pull/5249/commits/b946bbfa895fc49678725bad5f95e0d20a918c6d)